wsd/ClientRequestDispatcher.cpp

```diff
@@ -763,8 +763,6 @@ void ClientRequestDispatcher::handleIncomingMessage(SocketDisposition& dispositi
 
                     launchAsyncCheckFileInfo(_id, accessDetails, RequestVettingStations);
                 }
-
-                socket->shutdown();
```

Comment:
cypress tests pass/fail with this shutdown present/omitted

Reply:
using:

if (request.getMethod() == Poco::Net::HTTPRequest::HTTP_POST)
    socket->shutdown();

here I think worksaround the problem, which I think is that ClientRequestDispatcher::handleIncomingMessage is using socket->eraseFirstInputBytes(map); to remove consumed data, but in the post we have body content that isn't consumed by that and gets left behind